.github/workflows/SpellCheck.yml

```
@@ -10,4 +10,4 @@ jobs:
       - name: Checkout Actions Repository
         uses: actions/checkout@v3
       - name: Check spelling
-        uses: crate-ci/typos@v1.14.5
+        uses: crate-ci/typos@v1.14.6
```

I think if one uses
```suggestion
        uses: crate-ci/typos@v1
```
you will only get these update PRs for major versions.

---

Yep, that's a good idea.